viewTask.php

```diff
@@ -52,7 +52,7 @@
     </tr>
   </tbody>
 </table>
-<form action="listView.php" onclick="window.top.location = 'https://mansci-db.uwaterloo.ca/~wmmeyer/wmmeyer/listView.php?r=t'" class="form-newList" method="POST">
+<form action="listView.php" onclick="window.top.location = 'https://mansci-db.uwaterloo.ca/~wmmeyer/r2/studentplanner/listView.php?r=t'" class="form-newList" method="POST">

```

Comment:
Confirm the URL with other members - these were reverted in https://github.com/MayyarAA/studentplanner/pull/39/files